golem-test-framework/src/config/mod.rs

```diff
@@ -54,6 +56,7 @@ pub trait TestDependencies {
         self.rdb().kill();
         self.redis_monitor().kill();
         self.redis().kill();
+        self.cassandra().kill();
```

Comment:
now that cassandra is lazy, will this boot up cassandra just to kill it?
if so, i think we should use RwMutex Option instead of lazy cell